client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -224,22 +227,89 @@ const getPresalesChatKey = ( responseCart: ObjectWithProducts ) => {
 };
 
 /* Include a condition for your use case here if you want to show a specific nudge in the checkout sidebar */
-function CheckoutSidebarNudge( { responseCart }: { responseCart: ResponseCart } ) {
+function CheckoutSidebarNudge( {
+	responseCart,
+	siteId,
+	formStatus,
+	changeSelection,
+	addItemToCart,
+	isCartPendingUpdate,
+	areThereDomainProductsInCart,
+}: {
+	responseCart: ResponseCart;
+	siteId: number | undefined;
+	formStatus: FormStatus;
+	changeSelection: OnChangeItemVariant;
+	addItemToCart: ( item: MinimalRequestCartProduct ) => void;
+	isCartPendingUpdate: boolean;
+	areThereDomainProductsInCart: boolean;
+} ) {
 	const isWcMobile = isWcMobileApp();
 	const isDIFMInCart = hasDIFMProduct( responseCart );
 	const hasMonthlyProduct = responseCart?.products?.some( isMonthlyProduct );
 	const shouldUseCheckoutV2 = useCheckoutV2() === 'treatment';
+	const isPurchaseRenewal = useMemo(
```

Comment:
Not a blocker but you don't need to memoize this; calculating it is not expensive and it's a boolean so using it as a reference anywhere is stable even across re-renders (true and false are always unchanged unless they actually change).

Reply:
I think it was just an unnecessary optimization there too.